go/kbfs/data/file_data.go

```diff
@@ -976,8 +976,9 @@ func (fd *FileData) Split(ctx context.Context, id tlf.ID,
 				// TODO: If we're down to just one leaf block at this
 				// level, remove the layer of indirection (KBFS-1824).
 				iptrs := pblock.IPtrs
-				pblock.IPtrs =
-					append(iptrs[:pb.childIndex], iptrs[pb.childIndex+1:]...)
+				pblock.IPtrs = make([]IndirectFilePtr, len(iptrs)-1)
```

Comment:
The linter complains when you assign the append result to a different variable than the first parameter to append.  Presumably because of the weird interactions with append and memory allocation?